src/platform/forms-system/src/js/utilities/data/reduceErrors.js

```diff
@@ -0,0 +1,225 @@
+// min/max length or item errors may show up as duplicates
+const errorExists = (list, name, index) =>
+  list.some(obj => obj.name === name && obj.index === index);
+
+// Keys to ignore within the pageList objects & pageList schema
+const ignoreKeys = [
+  'title',
+  'path',
+  'depends',
+  'schema',
+  'chapterTitle',
+  'chapterKey',
+  'pageKey',
+  'type',
+  'required',
+  'initialData',
+];
+
+/**
+ * @typedef Form~pageList
+ * @type {object[]}
+ * @property {string} title - page title
+ * @property {string} path - url path
+ * @property {object} uiSchema - page uiSchema
+ * @property {object} schema - page schema
+ * @property {string} chapterTitle - title shown in review accordion
+ * @property {string} chapterKey - chapter key from config/form
+ * @property {string} pageKey - page key within chapter key from config/form
+ */
+/**
+ * Find the chapter and page name that contains the property (name) which is
+ * then used to find the associated accordion on the review & submit page; the
+ * associated chapter (accordion) will then be highlighted to indicate a problem
+ * Eventually, the page name will be used to target the specific page, then form
+ * element to take the user directly to the problem
+ * @param {Form~pageList} pageList - list of all form pages from `route.pageList`
+ * @param {string} name - target end path where the error occurred
+ * @param {string} instance - path inside the page object within the uiSchema,
+ *   but we can't specify that since this is a recursive search function
+ * @return {object} - single page from the matching pageList object, or an empty
+ *   object if the name is not found on any page
+ */
+export const getPropertyInfo = (pageList = [], name, instance = '') => {
+  const findPageIndex = (obj, insideInstance = instance === '') => {
```

Comment:
Is `obj` a `page` or `uiSchema` or what?

Reply:
```
 * @return {object} - single page from the matching pageList object, or an empty
 *   object if the name is not found on any page
```

It's returning a `page` object, but if a matching page isn't found, it'll return an empty object (page)